sse-component/src/main/java/org/wso2/identity/oidc/sse/api/stream/StreamController.java

```diff
@@ -0,0 +1,381 @@
+                                           @RequestHeader(value = AUTHORIZATION) String accessToken) {
+
+        if (!validateAccessToken(accessToken)) {
+            return new ResponseEntity<>(null, HttpStatus.UNAUTHORIZED);
+        }
+        String id = getClientId(accessToken);
+        Optional<Stream> stream = streamRepository.findById(id);
+
+        if (stream.isPresent()) {
+            Stream streamSelect = stream.get();
+            List<Subject> newSubjects = new ArrayList<>();
+            try {
+                newSubjects.addAll(streamSelect.getSubjects());
+            } catch (Exception e) {
+
+            }
+            for (int i = 0; i < newSubjects.size(); i++) {
+                if (StringUtils.equals(removeSubject.getFormat(), newSubjects.get(i).getFormat())) {
+                    if (StringUtils.equals(removeSubject.getEmail(), newSubjects.get(i).getEmail())) {
+                        newSubjects.remove(i);
+                        streamSelect.setSubjects(newSubjects);
+                        streamRepository.save(streamSelect);
+                        return new ResponseEntity<>(HttpStatus.NO_CONTENT);
+                    }
+                }
+            }
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+        }
+        return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+    }
+
+    /**
+     * Verify if the stream is working properly.
+     *
+     * @param accessToken Authorization token.
+     * @return Stream configurations.
+     */
+    @PostMapping("verify")
+    @ApiOperation(value = "", notes = "Remove existing subject from the stream")
+    public ResponseEntity<? extends Object> verification(@RequestHeader(value = AUTHORIZATION) String accessToken) {
+
+
+        if (!validateAccessToken(accessToken)) {
+            return new ResponseEntity<>(null, HttpStatus.UNAUTHORIZED);
+        }
+
+        String id = getClientId(accessToken);
+        Optional<Stream> stream = streamRepository.findById(id);
+
+        if (stream.isPresent()) {
+            Stream getStream = new Stream(stream.get().getIss(), stream.get().getAud(),
+                    stream.get().getEventsDelivered());
+            return new ResponseEntity<>(getStream, HttpStatus.OK);
+        }
+        return new ResponseEntity<>(null, HttpStatus.NOT_FOUND);
+    }
+
+    /**
+     * Validate access token and get id
+     *
+     * @param accessToken Authorization token.
+     * @return stream id
+     */
+    private Boolean validateAccessToken(String accessToken) {
+
+        if (accessToken.isEmpty()) {
+            return false;
+        }
+        return true;
+    }
+
+    /**
+     * Get client Id from authorization token.
+     *
+     * @param accessToken Authorization token.
+     * @return Client Id
+     */
+    private String getClientId(String accessToken) {
+
+        String requestBody = Constants.TOKEN + "=" + accessToken.substring(7);
```

Comment:
what this access token is, why we are getting substring of accestoken? 
Can you explain what it requestbody is?